packages/api/tests/lambdas/test-sf-semaphore-down.js

```diff
@@ -0,0 +1,273 @@
+'use strict';
+
+const test = require('ava');
+const {
+  concurrency: {
+    Semaphore
+  },
+  aws,
+  testUtils: {
+    randomId,
+    randomString
+  }
+} = require('@cumulus/common');
+const { Manager } = require('../../models');
+const handler = require('../../lambdas/sf-semaphore-down');
+
+const createSnsWorkflowMessage = ({
+  status,
+  priorityInfo
+}) => ({
+  Sns: {
+    Message: JSON.stringify({
+      cumulus_meta: {
+        execution_name: randomString(),
+        priorityInfo
+      },
+      meta: {
+        status
+      }
+    })
+  }
+});
+
+let manager;
+
+const setSemaphoreValue = async (key, value) =>
+  aws.dynamodbDocClient().put({
+    TableName: process.env.SemaphoresTable,
+    Item: {
+      key,
+      semvalue: value
+    }
+  }).promise();
+
+test.before(async () => {
+  process.env.SemaphoresTable = randomId('semaphoreTable');
+  manager = new Manager({
+    tableName: process.env.SemaphoresTable,
+    tableHash: { name: 'key', type: 'S' }
+  });
+  await manager.createTable();
+});
+
+test.beforeEach(async (t) => {
+  t.context.semaphore = new Semaphore(
+    aws.dynamodbDocClient(),
+    process.env.SemaphoresTable
+  );
+});
+
+test.after.always(async () => {
+  await manager.deleteTable();
```

Comment:
No need for `async`/`await`, just returns a promise.
`test.after.always(() => manager.deleteTable())`